ubports/system-image-upgrader

```diff
@@ -279,7 +279,7 @@ USE_SYSTEM_PARTITION=1
 # However, do not use the block device name in the kernel command line, as that is not consistently
 # named in booting normal and recovery modes. Expect fstab to have a system mountpoint or use a fallback.
 if [ "$USE_SYSTEM_PARTITION" -eq 1 ];then
-    SYSTEM_PARTITION=$(grep "^[^#]" /etc/fstab | grep -e "\(/mnt\)*/system\(_root\)*" | cut -f 1 -d\ )
+    SYSTEM_PARTITION=$(grep "^[^#]" /etc/fstab | grep -v "/system_ext" |  grep -e "\(/mnt\)*/system\(_root\)*" | cut -f 1 -d\ )
```

Comment:
Can you merge this with the check for system_root behind?

Reply:
Tried and failed, which is why I did the filtering in between.